CONTRIBUTING.md

```diff
@@ -75,23 +75,21 @@ How to edit OpenAPI file:
 -  If a 0-level model has dedicated sub-models, then those model's names should follow the pattern:
   `{resource}_{action}_{differentiator}_{request|response}_{body|param|query}`
   where the  `differentiator` describes the child model, e.g., `publication`.
-- If a model is used by more than one API endpoint (general model), we use simple domain language, e.g. `voucher`, `customer`, `product`, `discount`, `discount_unit`
+- If a model is used by more than one API endpoint (general model), we use simple domain language, e.g. `Customer`, `Category`, `Discount`, `Discount_unit`
```

Comment:
shouldn't it be DiscountUnit ?

Reply:
could you change the `Use snake case casing.` to `Use PascalCase casing.` in line 71?

Reply:
also :

```
- If a model is used as a specific API endpoint description (0-level model), then we follow the pattern: `{resource}_{action}_{request|response}_{body|param|query}`, where:
	  - `resource`: plural name taken from API path, e.g. `vouchers`, `customers`, `products`
	  - `action` : `get`(single record), `list`, `update`, `delete`, `create` (etc.)
-  If a 0-level model has dedicated sub-models, then those model's names should follow the pattern:
  `{resource}_{action}_{differentiator}_{request|response}_{body|param|query}`
  where the  `differentiator` describes the child model, e.g., `publication`.
  ```
 to :
 
 ```
 - If a model is used as a specific API endpoint description (0-level model), then we follow the pattern: `{Resource}{Action}{Request|Response}{Body|Param|Query}`, where:
	  - `Resource`: plural name taken from API path, e.g. `Vouchers`, `Customers`, `Products`
	  - `Action` : `Get`(single record), `List`, `Update`, `Delete`, `Create` (etc.)
-  If a 0-level model has dedicated sub-models, then those model's names should follow the pattern:
  `{Resource}{Action}{Differentiator}{Request|Response}_{Body|Param|Query}`
  where the  `Differentiator` describes the child model, e.g., `Publication`.
  ```